common/jmeter/oidc/SubOrg_OIDC_AuthCode_Redirect_WithConsent.jmx

```diff
@@ -0,0 +1,1933 @@
+          </elementProp>
+          <elementProp name="password" elementType="Argument">
+            <stringProp name="Argument.name">password</stringProp>
+            <stringProp name="Argument.value">Password_1</stringProp>
+            <stringProp name="Argument.metadata">=</stringProp>
+          </elementProp>
+          <elementProp name="scope" elementType="Argument">
+            <stringProp name="Argument.name">scope</stringProp>
+            <stringProp name="Argument.value">openid</stringProp>
+            <stringProp name="Argument.metadata">=</stringProp>
+          </elementProp>
+          <elementProp name="consumerKeyPrefix" elementType="Argument">
+            <stringProp name="Argument.name">consumerKeyPrefix</stringProp>
+            <stringProp name="Argument.value">b2b_consumerKey_</stringProp>
+            <stringProp name="Argument.metadata">=</stringProp>
+          </elementProp>
+          <elementProp name="consumerSecretPrefix" elementType="Argument">
+            <stringProp name="Argument.name">consumerSecretPrefix</stringProp>
+            <stringProp name="Argument.value">b2b_consumerSecret_</stringProp>
+            <stringProp name="Argument.metadata">=</stringProp>
+          </elementProp>
+          <elementProp name="callback_url_prefix" elementType="Argument">
+            <stringProp name="Argument.name">callback_url_prefix</stringProp>
+            <stringProp name="Argument.value">https://localhost/callback</stringProp>
+            <stringProp name="Argument.metadata">=</stringProp>
+          </elementProp>
+          <elementProp name="tenantDomainPrefix" elementType="Argument">
+            <stringProp name="Argument.name">tenantDomainPrefix</stringProp>
+            <stringProp name="Argument.value">dummytenant_</stringProp>
+            <stringProp name="Argument.metadata">=</stringProp>
+          </elementProp>
+          <elementProp name="subOrgNamePrefix" elementType="Argument">
+            <stringProp name="Argument.name">subOrgNamePrefix</stringProp>
+            <stringProp name="Argument.value">sub_org_</stringProp>
+            <stringProp name="Argument.metadata">=</stringProp>
+          </elementProp>
+        </collectionProp>
+      </Arguments>
+      <hashTree/>
+      <Arguments guiclass="ArgumentsPanel" testclass="Arguments" testname="User Defined Variables" enabled="true">
+        <collectionProp name="Arguments.arguments">
+          <elementProp name="noOfThreads" elementType="Argument">
+            <stringProp name="Argument.name">noOfThreads</stringProp>
+            <stringProp name="Argument.value">${__P(concurrency,1)}</stringProp>
+            <stringProp name="Argument.metadata">=</stringProp>
+          </elementProp>
+          <elementProp name="noOfSubOrgs" elementType="Argument">
+            <stringProp name="Argument.name">noOfSubOrgs</stringProp>
+            <stringProp name="Argument.value">${__P(subOrgCount,5000)}</stringProp>
```

Comment:
Having 5000 sub-organizations as the default value is too much I think. Isn't it an extreme case? 
Shall we have a common case default value here?

Reply:
@AnuradhaSK any thoughts?

Reply:
Thinking on a medium size B2B SaaS products usually number of customers will be 100-1k I guess.
But we have got few cases like having orgs 25k. We want to get performace number for this kind of higher number as well.
but as  a default value we can go with 1000

Reply:
If the value needs to be used when publishing results is 5000, then we should keep it as the default value and add an instruction on the readme to adjust the value as you require. @Sachin-Mamoru wdyt?

Reply:
Can't we parameterize this value and pass it through the Jenkins job as an additional parameter?

<img width="930" alt="image" src="https://github.com/wso2/performance-is/assets/59449070/895deff6-acbf-4f90-9eed-1c6c6ff5bf8a">

WDYT?

Reply:
If we are to parameterize this, then the variables such as user count should also be parameterize which can be used for other test scenarios as well. Therefore shall we onboard with another effort.